src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -42,6 +54,28 @@ const SignedChatScreen = () => {
     flatlistRef.current?.scrollToEnd();
   };
 
+  const moveChatToAnon = async () => {
+    try {
+      if (selectedChannel?.channelType === 'GROUP') {
+        ToastMessage.show({
+          type: 'asNative',
+          text1: 'Coming soon: Anonymous messages are not enabled yet within group chats',
+          position: 'bottom'
+        });
+        return;
+      }
+      setLoading(true);
+      await moveToAnonymousChannel({
+        oldChannelId: selectedChannel?.id,
+        targetUserId: memberChat.user_id
+      });
+    } catch (e) {
+      console.log('error moving chat to signed channel', e);
+    } finally {
+      setLoading(false);
+    }
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling could be improved here. Currently, you're just logging the error to the console. It would be more user-friendly to display an error message to the user when something goes wrong. You can use the `ToastMessage` component to show the error message.

```diff
- console.log('error moving chat to signed channel', e);
+ ToastMessage.show({
+   type: 'error',
+   text1: 'Error',
+   text2: 'There was an error moving the chat to the signed channel.',
+ });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->